.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature Request 🚀
+    url: https://firebase.google.com/support/contact/bugs-features/
```

Comment:
This will redirect feature requests to https://firebase.google.com/support/contact/bugs-features/. Thoughts?

Reply:
Am I supposed to be able to navigate to that link?

Also I'm kind of confused about what effect this config has.

Reply:
Ah I see, this is will present an alternative to creating an issue, nice.

Reply:
I'm wondering if the link should be: https://firebase.google.com/support/troubleshooter/report/features

Reply:
Ahh yeah looks like this link is incorrect. I'll replace it with this https://firebase.google.com/support/troubleshooter/report/features

If you look at the https://github.com/firebase/firebase-tools/issues/new/choose you you can see that the bottom two options are `contact_links`. Basically they will link the user Feature Request form on firebase.com when someone clicks the Open button.

Reply:
Sounds awesome